index.js

```diff
@@ -121,6 +122,17 @@ function convertCrontabs() {
 class ServerlessLocalCrontabs {
   constructor(serverless, options) {
     this.serverless = serverless;
+    if (this.serverless.configSchemaHandler) {
```

Comment:
Sorry for not mentioning it at first - I've just double checked and there can be a situation where the `configSchemaHandler` is defined, but the `defineFunctionEventProperties` method is not - let's also check for the existence of this method before trying to call it, okay? It was added in `2.11.0` so let's not break users that are on lower versions: https://github.com/serverless/serverless/blob/main/CHANGELOG.md#2110-2020-11-06

Reply:
@pgrzesik done